e2e-tests/tfm/cypress/e2e/journeys/utilisation-reports/view-utilisation.spec.js

```diff
@@ -119,4 +120,11 @@ context('Users can view utilisation', () => {
       cy.assertText(cy.get('[data-cy="fees-payable"]'), `${CURRENCY.GBP} 100.00`);
     });
   });
+
+  it('should be able to download report for offline viewing', () => {
```

Comment:
Can we say e.g "should render a link"  instead of "should be able to download" please, because (quite rightly) it's not testing the actual download, rather that the link exists